src/app/find-help/view-offer-transport/view-offer-transport.component.ts

```diff
@@ -38,6 +39,9 @@ export class ViewOfferTransportComponent implements OnInit {
 
   getTransportOffer() {
     this.transportResourceService.getTransport(this.offerId).subscribe((response) => {
+      if (response == null) {
```

Comment:
gdy nie ma ogłoszenia zwróci 404,
subscribe(
res =>
error => tutaj kierujemy, może wystarczy użyć relatywnej ścieżki ?
)

Reply:
I robimy kompleksowo, dla  3 kategorii

Reply:
Gdy wpisywałem nieistniejący id, to dostawałem null. Chyba muszę zmienić odpowiedź z mocków?

jakie trzy kategorie?